main/src/main/scala/sbt/Keys.scala

```diff
@@ -325,6 +325,7 @@ object Keys {
   val dependencyClasspathAsJars = taskKey[Classpath]("The classpath consisting of internal and external, managed and unmanaged dependencies, all as JARs.")
   val fullClasspathAsJars = taskKey[Classpath]("The exported classpath, consisting of build products and unmanaged and managed, internal and external dependencies, all as JARs.")
   val internalDependencyConfigurations = settingKey[Seq[(ProjectRef, Set[String])]]("The project configurations that this configuration depends on")
+  val classLoaderCloseMode = settingKey[ClassLoaderCloseMode]("Configures what do when a classloader is used after it has been closed.")
```

Comment:
As we discussed in the meeting, it might be better to stick to Boolean for setting so the build users can use machine-wide settings.

Reply:
Yeah. I've just been stuck on what to name them.

Reply:
```suggestion
  val allowZombieClassLoader = settingKey[Boolean]("Allows application to load classes after run or test task has finished; throws an error otherwise.")
```

I'd go with warning-or-strict option.

Reply:
Or maybe for sbt 1.3.1 it's `false` by default, like HTTP, and it lets you use zombie but just warns. By setting `ThisBuild / allowZombieClassLoader := true` you can silence the warning. So it's warning-or-ignore axis. And in some future version it becomes strict-or-ignore?

Reply:
Ok. I made that change.